tests/Cluster.py

```diff
@@ -1416,7 +1413,7 @@ def bootstrap(self, biosNode, totalNodes, prodCount, totalProducers, pfSetupPoli
             Utils.Print("ERROR: Issue verification failed. Excepted %s, actual: %s" %
                         (expectedAmount, actualAmount))
             return None
-
+            
```

Comment:
because of I neededscenario where I don't need bootstrap but need other nodes. In current developscenario cluster "prunes" other nodes leaving only bios.